w2/zadania/Bartman18/zadanie3.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Scanner;
+
+public class zadanie3 {
```

Comment:
![image](https://user-images.githubusercontent.com/50357817/224505432-4802ea94-30e0-48dd-813d-ecceea79c8b3.png)
jak to nie można :(

Reply:
Poprawione, powinno być już dobrze